src/wp-admin/includes/class-wp-privacy-policy-content.php

```diff
@@ -424,7 +424,7 @@ public static function privacy_policy_guide() {
 						<span class="screen-reader-text">
 							<?php
 							/* translators: %s: Plugin name. */
-							sprintf( __( 'Copy suggested policy text from %s.' ), $plugin_name );
+							echo sprintf( __( 'Copy suggested policy text from %s.' ), $plugin_name );
```

Comment:
Can just do `printf` :-)

```suggestion
							printf( __( 'Copy suggested policy text from %s.' ), $plugin_name );
```